RELEASING.md

```diff
@@ -0,0 +1,41 @@
+# Releasing
+
+The following artifacts are created as a result of releasing Fabric Chaincode Node:
+
+- docker images
+    - [fabric-nodeenv](https://hub.docker.com/r/hyperledger/fabric-nodeenv)
+- npm modules
+    - [fabric-contract-api](https://www.npmjs.com/package/fabric-contract-api)
+    - [fabric-shim](https://www.npmjs.com/package/fabric-shim)
+    - [fabric-shim-api](https://www.npmjs.com/package/fabric-shim-api)
+    - [fabric-shim-crypto](https://www.npmjs.com/package/fabric-shim-crypto)
+
+**Note:** A docker image with a matching V.R version is required before releasing a new version of Fabric.
```

Comment:
V2 only statement...

Reply:
Although it's not used for releases, I'm assuming master is the same. Is it V.R.M for when I add the release process to the 1.4 branch, or is the docker image not required? Is there even a docker image?

Reply:
For 1.4 it uses the `ccenv` rather than have it's own dedicated one.